components/CreateCandidateForm/CreateCandidate.module.scss

```diff
@@ -0,0 +1,37 @@
+﻿@import '../../pageStyles/constants';
+
+.subHeader{
+  text-align: center;
+  font-size: $desktopSubHeaderFontSize;
+  color: $softwireYellow;
+  font-family: $softwireFont;
+}
+
+.form{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.input{
+  color: $softwireBlack;
+  background-color: $softwireWhite;
+  border: solid 2px $softwireYellow;
+  font-family: $softwireFont;
+}
+
+.label{
+  color: $softwireYellow;
+  font-size: $contentFontSize;
+  font-family: $softwireFont;
+  margin-top: 10px;
+}
+
+.createButton{
+  background-color: $softwireYellow;
+  border: none;
+  padding: 10px 20px;
+  font-size: $buttonFontSize;
+  font-family: $softwireFont;
+  margin-top: 10px;
+}
```

Comment:
I'd guess almost all of this is shared with the login page?
If so we should create some mixins or components to share our styling between the 2.

Reply:
made some mixins

Reply:
probably would have been easier to just have a shared stylesheet as they're identical right now